src/main/webapp/chart.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A New Chart</title>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+	<script type="text/javascript">
+		//loads the corecharts package, defined in loader.js script above
+	    google.charts.load('current', {packages: ['corechart']});
```

Comment:
Another small nitpick: Scoot back the comment on line 7 so that it is in line with the rest of that block of code.